BasicCoding/BasicCoding/BasicExercises.playground/Contents.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+//Exercise 1:
+
+let distance: UInt = 41
+
+switch distance
+{
+    case 0:
+        print("Here")
+    case 0..<5:
+        print("Inmediate vicinity")
+    case 5...15:
+        print("Near")
+    case 15...40:
+        print("Kind of far")
+    default:
+        print("Far")
+}
+
+//Exercise 2
+
+func comparator(_ array: [Int], _ number: Int)
+{
+    var greaterCounter = 0
+    var lowerCounter = 0
+    for item in array
+    {
+        if item > number
+        {
+            greaterCounter += 1
+        }else if item < number
```

Comment:
also check spacing between the bracket and de "else"

Reply:
Fixed. Space added after brackets